pagopa/io-app
# chore: [IOBP-985] Add onboarding payment method success mixpanel analytics event

## Short description
This PR adds a mixpanel analytics event dispatched when the user onboard successfully a payment method

## List of changes proposed in this pull request
- Added the `analytics` folder inside the `ts/features/payments/onboarding` sub-feature with its dispatcher method `trackSuccessOnboardingPaymentMethod`;
- Added the logic inside the `PaymentsOnboardingFeedbackScreen.tsx` to track the onboarded payment method selected;

## How to test
- Start the onboarding payment method flow from the Wallet screen;
- Check that when you complete the onboarding successfully the mixpanel event is emitted
